clustering.Rmd

```diff
@@ -663,11 +671,11 @@ penguin_clust_ks
 ```
 
 Then we use `rowwise` \index{rowwise} + `mutate` to apply the `kmeans` function 
-within each row, to each K. 
+within each row to each K. 
 However, given that the `kmeans` function 
-returns a model object to us, (not a vector),
+returns a model object to us (not a vector)
 we will need to store the results as a list columm.
-This can work because lists are super vectors, and so they are legitimate 
+This can work because both vector and lists are legitimate 
```

Comment:
"model object to us (not a vector)" -> "model object to us (not a vector),"    (added comma at the end)

"This can work" -> "This works"

"both vector" -> "both vectors"

Reply:
fixed